app/views/products/_image_detail.html.erb

```diff
@@ -30,4 +30,4 @@
   </div>
 
 </div>
-<span class="featured"><%= @product.label.short_title %></span>
+<!-- <span class="featured"><%= @product.label.short_title %></span> -->
```

Comment:
Đoạn này ko dùng à Toàn ơi, có thể bỏ đi không thế

Reply:
@vuducluan : ok a, đã bỏ